CONTRIBUTING.rst

```diff
@@ -31,7 +31,7 @@ To build the docs, you must first create a development environment (see
 below) and then in the ``docs/`` directory run::
 
     $ cd docs
-    $ make html
+    $ sphinx-build -b html source build
```

Comment:
@godslayer201 
Make this `sphinx-build -b html source build/html`.

Later, in the same file, change `python -m http.server`
to `python -m http.server 1000 --bind 127.0.0.1`, as `python -m http.server` can lead to `OSError: [WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions` on Windows (It's not a UAC issue. Something's broken in `http.server`).

Finally, change "And point your browser to http://0.0.0.0:8000." to "And point your browser to http://127.0.0.1:1000.".

Reply:
Understood. 

Will update my PR.

Reply:
Hey @godslayer201 Any updates on this?

Reply:
Hey Shreyas,

I'm not working on it at the moment.

I would finish the PR by Friday